Resources/Prototypes/_Harmony/Catalog/Jukebox/standard.yml

```
@@ -0,0 +1,6 @@
+﻿
+- type: jukebox
```

There is an extra space here
```suggestion
- type: jukebox
```